source/developing.html.erb.md

```diff
@@ -175,8 +175,7 @@ Use structural elements to mark-up headings, lists, tables, etc. HTML5 provides
 {:.attach_permalink}
 ## Help users avoid and correct mistakes
 
-
-Be as forgiving of format as possible when processing user input. For example, accept phone numbers that include spaces and delete the spaces as needed. Clearly identify errors that cannot be auto-corrected. Provide an in-page link for easy access from the error to the field. Work with content writers to agree on content for instructions and messages.
+Be as forgiving of format as possible when processing user input. For example, accept phone numbers that include spaces and delete the spaces as needed. Clearly identify errors that cannot be auto-corrected. Work with content writers to agree on content for instructions and messages.
```

Comment:
Agreed to leave the line out as there are plenty of other items that are equally simple and helpful